sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -105,6 +105,7 @@ export interface AppendBlobAppendBlockFromUrlHeaders {
     contentMD5?: Uint8Array;
     date?: Date;
     encryptionKeySha256?: string;
+    encryptionScope?: string;
```

Comment:
>encryptionScope?: string; [](start = 4, length = 25)

Why encryptionScope parameter is added in PR about incremental snapshot?

Reply:
Comes directly from the generated code. No easy way to isolate the swagger generated changes.